vars/createArgusTestRun.groovy

```diff
@@ -2,15 +2,14 @@
 
 def call(Map params) {
     def test_config = groovy.json.JsonOutput.toJson(params.test_config)
-    def cloud_provider = getCloudProviderFromBackend(params.backend)
     sh """#!/bin/bash
         set -xe
 
         echo "Creating Argus test run ..."
         if [[ -n "${params.requested_by_user}" ]] ; then
             export BUILD_USER_REQUESTED_BY=${params.requested_by_user}
         fi
-        export SCT_CLUSTER_BACKEND="${cloud_provider}"
+        export SCT_CLUSTER_BACKEND="${params.backend}"
```

Comment:
This file change reverting must have been done in the proper commit.

Reply:
Reverted in the proper commit.